requirements.txt

```diff
@@ -54,14 +65,21 @@ python-dateutil==2.8.1
 pytz==2021.1
 pyzmq==22.0.3
 requests==2.20.1
+retrying==1.3.3
```

Comment:
Are we retrying some of the code???

Reply:
It's a dependency of plotly, which is in turn a dependency of pymatgen